js/form_editor_controller.js

```diff
@@ -360,8 +387,6 @@ class GlpiFormEditorController
      * Must be executed after each actions.
      */
     computeState() {
-        let global_q_index = 0;
```

Comment:
If this index question is not longer global, we lose questions when saving the form.

For exemple, the first question of every section will have the same name "_questions[0]", which mean that only one of these questions will be saved and the others will be lost.